### ISSUE TITLE
AVT 1 - Skeleton: Checkbox, NumberInput, Select, Slider, TextArea & TextInput DAP violation for missing label

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4176]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

Detailed Description
Go to the React skeleton for Checkbox, NumberInput, Select, Slider, TextArea & TextInput
Run DAP and the following issue is found on the on each skeleton component (see screenshots):
<img width="1310" alt="checkbox" src="https://user-images.githubusercontent.com/21676914/66066760-bfffe700-e50f-11e9-9808-a4837a9ca3a9.png">
<img width="1278" alt="numberinput" src="https://user-images.githubusercontent.com/21676914/66066767-c4c49b00-e50f-11e9-8770-93876c2863eb.png">
<img width="1330" alt="select" src="https://user-images.githubusercontent.com/21676914/66066783-cee69980-e50f-11e9-9432-5b1ab2096352.png">
<img width="1323" alt="TextArea" src="https://user-images.githubusercontent.com/21676914/66066805-ddcd4c00-e50f-11e9-978f-adfb6b0ad0fa.png">
<img width="1301" alt="TextInput" src="https://user-images.githubusercontent.com/21676914/66066806-ddcd4c00-e50f-11e9-91d0-2fb3d7042420.png">
